README.md

```diff
@@ -1,8 +1,8 @@
 # Check Digit Currency Library
 
-The Check Digit currency library is the officially sanctioned method for Check Digit services to deal with currency types, formatting and country/currency relationships. Features:
+The Check Digit currency library is the officially sanctioned method for Check Digit services to deal with currency types, formatting and country/currency relationships at a particular date/time. Features:
 
-- various currency and country lookup functions
+- various currency and country lookup functions at a particular date/time
```

Comment:
i think we need to be careful about the wording of this and how it works. this functionality will not return accurate information for any point in time passed in. we should probably have a start date for the start of the data that we have at Check Digit (maybe 2018 and beyond). anything earlier than that should probably throw an error as we will not provide accurate data earlier (obtaining the data for France from 1991 would return francs as the currency and not euros for example).

Reply:
I have rephrased it as below. Let me know if it is ok.
- various currency and country lookup functions at a particular date/time starting 2018 and beyond, anything earlier than that would probably throw an error as we will not provide accurate data earlier.

Reply:
that's fine. you'll of course need to enforce it in the code and make sure to have accurate data from 2018 and beyond.